wallet/src/account/output_cache/mod.rs

```diff
@@ -1692,3 +1747,20 @@ fn apply_total_supply_mutations_from_tx(
 
     Ok(total_supply)
 }
+
+fn uses_conflicting_nonce(
+    unconfirmed_tx: &WalletTx,
+    confirmed_account_type: AccountType,
+    confirmed_nonce: AccountNonce,
+) -> bool {
+    unconfirmed_tx.inputs().iter().all(|inp| match inp {
+        TxInput::Utxo(_) => true,
+        TxInput::AccountCommand(nonce, cmd) => {
+            confirmed_account_type == cmd.into() && *nonce <= confirmed_nonce
+        }
+        TxInput::Account(outpoint) => {
+            confirmed_account_type == outpoint.account().into()
+                && outpoint.nonce() <= confirmed_nonce
+        }
+    })
+}
```

Comment:
I am a bit confused on this, shouldn't it be any instead of all, and false for Utxo?
If all inputs are Utxos this will return that it is a conflicting nonce tx?

Reply:
you are right thanks